js/dlg/message-box.js

```diff
@@ -102,6 +111,37 @@ messageBox.show = async ({
       scroll() {
         scrollTo(messageBox._blockScroll.x, messageBox._blockScroll.y);
       },
+      mouseDown(event) {
+        if (event.button !== 0) {
+          return;
+        }
+        if (!listening) {
+          window.on('mouseup', messageBox.listeners.mouseUp, {passive: true});
+          window.on('mousemove', messageBox.listeners.mouseMove, {passive: true});
+          listening = true;
+        }
+        clickX = event.clientX - offsetX;
+        clickY = event.clientY - offsetY;
+      },
+      mouseUp(event) {
+        if (event.button !== 0) {
+          return;
+        }
+        window.off('mouseup', messageBox.listeners.mouseUp);
+        window.off('mousemove', messageBox.listeners.mouseMove);
+        listening = false;
+      },
+      mouseMove(event) {
+        const x = event.clientX - clickX;
+        const y = event.clientY - clickY;
+
+        offsetX = x;
+        offsetY = y;
+
+        $('#message-box > div').style.transform =
+          `translateX(${clamp(x, -30, 30)}px) 
```

Comment:
No, that's not what I meant... Now I can't drag the box for more than 30px at once.

Reply:
Something like event.clientX > 30 && event.clientX < innerWidth - 30 && event.clientY > 30 && event.clientY < innerHeight - 30

Reply:
Or rather x = clamp(event.clientX, 30, innerWidth - 30), y = clamp(event.clientY, 30, innerHeight - 30)

Reply:
The clamping should do the job for now.

Reply:
Sorry for the misunderstanding.